src/lib/helpers.ts

```diff
@@ -230,3 +231,12 @@ export const extractNodes = <Node extends object, T = Node>(
 export const isInteger = (str: string) => {
   return Number.isInteger(parseFloat(str))
 }
+
+// For some users with no favourites, Gravity can return a 404 (which reflects the
```

Comment:
kept the 🇬🇧 spelling of this comment (possibly all 🍝  from @orta but I wouldn't want to _stop_ catching something we were for mobile app w/o more proper logging  + monitoring, so that's for another time - whether or not this comment still applies at all)